pkg/grizzly/workflow.go

```diff
@@ -74,61 +76,57 @@ func List(config Config, resources Resources) error {
 	return w.Flush()
 }
 
-func getPrivateElementsScript(jsonnetFile string, handlers []Handler) string {
-	const script = `
-    local src = import '%s';
-    src + {
-    %s
-    }
-	`
-	handlerStrings := []string{}
-	for _, handler := range handlers {
-		for _, jsonPath := range handler.GetJSONPaths() {
-			handlerStrings = append(handlerStrings, fmt.Sprintf("  %s+::: {},", jsonPath))
-		}
-	}
-	return fmt.Sprintf(script, jsonnetFile, strings.Join(handlerStrings, "\n"))
-}
+//go:embed grizzly.jsonnet
+var script string
 
 // Parse evaluates a jsonnet file and parses it into an object tree
 func Parse(config Config, jsonnetFile string, targets []string) (Resources, error) {
 
-	script := getPrivateElementsScript(jsonnetFile, config.Registry.Handlers)
+	script := fmt.Sprintf(script, jsonnetFile)
```

Comment:
In the Tanka project we noticed the embedded Jsonnet has some awkward side effects: https://github.com/grafana/tanka/issues/522

I'm not sure if this applies here, but if we are going to use top level arguments, we might not want to make the same mistake.

Reply:
That's fair, but at present Grizzly doesn't offer any `tla` stuff. I'd expect that, over time, we'll move away from the snippet approach, which is needed because of hidden values anyway (so we already have the problem). This allows us to move towards a place where we don't use them at all (only consuming k8s style resources).